lib/databasedotcom/flow.rb

```diff
@@ -0,0 +1,25 @@
+module Databasedotcom
+  # A Salesforce API endpoint that can trigger predefinted flows in salesforce to kick off salesforce side processes.
+  # A flow can be triggered by its unique developer name and passed a hash of input variables.
+
+  class Flow
+    def initialize(client, flow_name, params = {}, options = {})
+      @client = client
+      @flow_name = flow_name
+      @params = params
+      @version = options[:version] || '32.0'
+    end
+
+    def run
+      params_hash = {inputs: [@params]}
+      @client.http_post(flow_url, params_hash.to_json)
+    end
+
+    protected
+
+    def flow_url
+      "/services/data/v#{@version}/actions/custom/flow/#{@flow_name}"
```

Comment:
Hmm good point. I took a look at the docs and flow names cannot contain spaces in salesforce and are automatically separated by underscores. The is considered the API name for the flow so they handle validation on creation in their UI.